.dotnet/src/Custom/FineTuning/FineTuningJob.cs

```diff
@@ -0,0 +1,23 @@
+using System.ClientModel.Primitives;
+using System.Collections.Generic;
+using System.Text.Json;
+
+namespace OpenAI.FineTuning;
+
+[CodeGenModel("FineTuningJob")]
+public partial class FineTuningJob
+{
+    internal static FineTuningJob FromResponse(PipelineResponse pipelineResponse)
+    {
+        using var document = JsonDocument.Parse(pipelineResponse.Content);
+        return DeserializeFineTuningJob(document.RootElement);
+    }
+
+    public FineTuningJobStatus Status { get; }
+
+    public string Object { get; }
+
+    public FineTuningJobHyperparameters? Hyperparameters { get; set; } 
+    internal IReadOnlyList<InternalFineTuningIntegration> Integrations { get; }

```

Comment:
When we expose this, we'll probably want to make it a list of "FineTuningIntegrationConfiguration" instances or similar. I know this part is WIP, though!

Reply:
Can't come up with a clean way to handle this. Let's discuss it when there's time. To achieve a simple syntax like this: 

```c
FineTuningJob job = client.CreateJob(
  "gpt-3.5-turbo",
  sampleFile.Id,
  options: new() { Integrations = [new IntegrationWandB("ft-tests")] }
);
```
pretty hacky overrides of getters and setters are needed